sqrl-pytorch/src/main/java/com/datasqrl/pytorch/EmbeddingsService.java

```diff
@@ -0,0 +1,43 @@
+package com.datasqrl.pytorch;
+
+import ai.djl.MalformedModelException;
+import ai.djl.huggingface.translator.TextEmbeddingTranslatorFactory;
+import ai.djl.inference.Predictor;
+import ai.djl.repository.zoo.Criteria;
+import ai.djl.repository.zoo.ModelNotFoundException;
+import ai.djl.repository.zoo.ZooModel;
+import ai.djl.training.util.ProgressBar;
+import ai.djl.translate.TranslateException;
+
+import java.io.IOException;
+
+public class EmbeddingsService {
+
+    private ZooModel<String, float[]> model = null;
```

Comment:
Should this be a hashmap keyed by model name so users can use different embedding models in the same SQRL script. Right now, it'll use the first model and ignore all others.